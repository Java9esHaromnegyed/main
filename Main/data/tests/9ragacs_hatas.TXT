loadArena testArena9.txt
initArena
tick 2