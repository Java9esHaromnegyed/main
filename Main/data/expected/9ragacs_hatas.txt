arenaLoaded src: testArena9.txt
robotAdded id: 1 name: „player1” pos: [0; 0]
obstacleAdded type: PUTTY pos: [1; 0]
arenaInited
tick
robotMoved id: 1 pos: [1; 0]
robotSlowedDown id: 1 speed: 0.5
tick
robotMoved id: 1 pos: [2.5; 0]